runtime/mainnet/src/config/governance.rs

```diff
@@ -0,0 +1,424 @@
+use frame_support::{
+	parameter_types,
+	traits::{EitherOfDiverse, NeverEnsureOrigin},
+	weights::Weight,
+};
+use frame_system::EnsureRoot;
+use pallet_collective::EnsureProportionAtLeast;
+use parachains_common::BlockNumber;
+use pop_runtime_common::DAYS;
+
+use crate::{
+	AccountId, Council, Runtime, RuntimeBlockWeights, RuntimeCall, RuntimeEvent, RuntimeOrigin,
+};
+
+type UnanimousCouncilVote = EitherOfDiverse<
+	EnsureRoot<AccountId>,
+	EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 1>,
+>;
+
+type AtLeastThreeFourthsOfCouncil = EitherOfDiverse<
```

Comment:
Minor: would personally group the type aliases in a block (no space) and sort alphabetically so its easier to find things.